src/System.Text.Json/src/System/Text/Json/JsonTokenType.cs

```diff
@@ -15,17 +15,68 @@ public enum JsonTokenType : byte
     {
         // Do not re-order.
         // We rely on the ordering to quickly check things like IsTokenTypePrimitive
+
+        /// <summary>
+        ///   Indicates that there is no value (as distinct from <see cref="Null"/>).
+        /// </summary>
+        /// <remarks>
+        ///   This is the default token type if no data has been read by the <see cref="Utf8JsonReader"/>.
+        /// </remarks>
         None,
+
+        /// <summary>
+        ///   Indicates that the token type is the start of a JSON object.
+        /// </summary>
         StartObject,
+
+        /// <summary>
+        ///   Indicates that the token type is the end of a JSON object.
+        /// </summary>
         EndObject,
+
+        /// <summary>
+        ///   Indicates that the token type is the start of a JSON array.
+        /// </summary>
         StartArray,
+
+        /// <summary>
+        ///   Indicates that the token type is the end of a JSON array.
+        /// </summary>
         EndArray,
+
+        /// <summary>
+        ///   Indicates that the token type is a JSON property name.
+        /// </summary>
         PropertyName,
+
+        /// <summary>
+        ///   Indicates that the token type is a JSON string.
+        /// </summary>
         String,
+
+        /// <summary>
+        ///   Indicates that the token type is a JSON number.
+        /// </summary>
         Number,
+
+        /// <summary>
+        ///   Indicates that the token type is the JSON literal <c>true</c>.
```

Comment:
Reserved keywords like `null`, `true` and `false` should be enclosed by `<see langword="keyword" />`:

```suggestion
        ///   Indicates that the token type is the JSON literal <see langword="true" />.
```

Reply:
If you are referring to a special object defined for json and not to the reserved keyword, then use see cref.

Reply:
This is not referring to the language key word but rather the JSON literals. We have similar documentation for `JsonValueType`: https://github.com/dotnet/corefx/blob/1a60edc784c89a8370d1f9539944f5c95261e7a8/src/System.Text.Json/src/System/Text/Json/Document/JsonValueType.cs#L38

What would `<see cref="true"/>` link to? I am not sure if that makes sense either, but I am open to suggestions.

I am leaning towards keeping this as is for now and fix both places once a decision is made:
https://github.com/dotnet/dotnet-api-docs/blob/85b7af914b69213a8e7ef7febacf2f9a5a99a13e/xml/System.Text.Json/JsonValueType.xml#L157

cc @bartonjs

Reply:
Also cc @rpetrusha if you have a suggestion here.

Reply:
In Docs, see langword is used so that reserved keywords are shown correctly depending on the programming language. The default is C#, but if you switch the documentation to F# or VB, you should be able to see the reserved keyword in its native language.

`<c>keyword</c>` does not translate to anything, and the Docs team will request me to change it to see langword anyway.

@rpetrusha can confirm or correct my statement.

Reply:
This isn't C# true, or VB True, it's the JSON literal value `true`.  So it shouldn't be a langword (it shouldn't capitalize when switching to VB).

Reply:
Apologies. Didn't read Ahson's comment correctly. We still need to do something about `<c>`, because I don't think that has a meaning in Docs xmls.

@ahsonkhan feel free to check it in. When the time comes for me to port it to Docs, the Docs team will address this. I'll mention you there as FYI for future reference. Sounds good?